when cross compiling, this conftest falls back to =cross when cross, and then
matches this case for what to do- so make it match all of linux for sysv instead
of aborting in the other branch for our -linux-musl triple.
--
--- a/configure.ac
+++ b/configure.ac
@@ -55,7 +55,7 @@
 
   if test $ac_cv_use_ipc = cross; then
     case "$host_os" in
-      (linux-gnu*)
+      (linux-*)
         ac_cv_use_ipc=sysv
         AC_MSG_RESULT([cross, guessing yes])
       ;;
